actionview/test/template/form_helper_test.rb

```diff
@@ -168,7 +168,8 @@ def url_for(object)
     @url_for_options = object
 
     if object.is_a?(Hash) && object[:use_route].blank? && object[:controller].blank?
-      object.merge!(controller: "main", action: "index")
+      object[:controller] = "main"
```

Comment:
@oniofchaos please use `"` only for interpolation

Reply:
That's the style I use in my own projects but it's not the style for the Rails codebase. You can see the Rubocop failures here: https://codeclimate.com/github/rails/rails/pull/32381